roles/mkthordocker/defaults/main.yml

```diff
@@ -0,0 +1,33 @@
+---
+# name: influxdb
+# docker_hub_image: influxdb:latest
+# description: "InfluxDB Container"
+# lvm:
+#   - name: influxdb
+#     size: 200G
+#     vg: 12tb_2020
+#     user: influxdb
+#   - name: influxdb
+#     size: 200G
+#     vg: 12tb_2020
+#     user: influxdb
+#   - name: influxdb2
+#     size: 200G
+#     vg: 12tb_2020
+#     user: influxdb
+# docker_env:
+#   PUID: 454
+#   GUID: 451
+#   INFLUXDB_HTTP_AUTH_ENABLED: true
+#   INFLUXDB_ADMIN_USER: admin
+#   INFLUXDB_ADMIN_PASSWORD: "{{ lookup('onepassword', 'HKRoad Ansible Passwords', section='THOR', field='influxdb admin pass') }}"
+# docker_ports:
+#   8086: 8086
+#docker_vols:
+  # "/etc/passwd": "/etc/passwd"
+  # "/etc/group": "/etc/group"
+  # "/mnt/influxdb/influxdb2": "/var/lib/influxdb2"
+  # "/mnt/influxdb/influxdb2-config": "/etc/influxdb2"
+# docker_options: |
+  # --user=454:451 \
+  # --ulimit nofile=50000:50000 \
```

Comment:
There's a setting in Code to "automatically add trailing newlines" or something along those "lines" :smile: